Frameworks/BlockchainCertificates/Signature.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Signature.swift
+//  cert-wallet
+//
+//  Created by Kim Duffy on 4/10/17.
+//  Copyright © 2017 Digital Certificates Project. All rights reserved.
+//
+
+import Foundation
+
+public struct Signature {
+    public let value : String?
+    public let created : String?
+    public let creator: String?
+
+    
+    public init(value: String?, created: String?, creator: String?) {
```

Comment:
You get this constructor for free! I'm pretty sure you can remove it & still use it.